### PR TITLE
chore: change favicon crown to updated crown icon

### PR DESCRIPTION
**Pull request by BenSurgisonPINS**
## Describe your changes

APPLICS-226 - https://pins-ds.atlassian.net/browse/APPLICS-226

This will require anyone developing locally to re-run the build prior to running the containers in docker.

## Useful information to review or test

- Removed all images from the public images folder for the Queen Elizabeth the 2nd crown.
- Added all the images for the King Charles the 3rd crown to the the public images folder sourced from govuk-frontend version 5.
- Added the headIcons block to the default layout to make sure the correct crown icons are used. 

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/views/layouts/default.njk [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 {%- set initiateScriptsConfig = [{ src: "/public/scripts/index.script.js" }] -%}
 
+{% set imagePath = '/public' %}
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/views/layouts/default.njk:7]: As this is the image path could we include the image folder: '/public/image' or change it to publicPath to accommodate manifest.json?

File: packages/forms-web-app/src/views/layouts/default.njk [hunk 2]
```diff
@@ -12,6 +14,15 @@
 	{{ pageTitle }}
 {% endblock %}
 
+{# Note this block can be removed as well as the relevent images in /public/images when updating to v5 of the govuk-frontend node module #}
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/views/layouts/default.njk:17]: Just a suggestion but might be nice to put this chunk into an includes named 'head-icons.njk'.

File: packages/forms-web-app/src/views/layouts/default.njk (hunk 1 above)
Comment (BenSurgisonPINS) [reply packages/forms-web-app/src/views/layouts/default.njk:7]: Yes, I think I'll go with your second suggestion

File: packages/forms-web-app/src/views/layouts/default.njk (hunk 2 above)
Comment (BenSurgisonPINS) [reply packages/forms-web-app/src/views/layouts/default.njk:17]: To be honest, I think it's more obvious here with the comment that it needs to be removed when the upgrade to govuk-frontend V5 is implemented.  The additional file would need to be deleted.  I will implement as you have described if you think it still makes sense for it to be in a separate file.

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**